_meeting-notes/10-10-2019.md

```diff
@@ -0,0 +1,27 @@
+# Sync Oct 10, 2019
+
+## Attendees
+
+- David Dias
+- Jim Pick
+
+## Notes
+
+- AI(David): create PR to list the current team and roles 
+  - Raul -> Architect 
+  - Jim
+  - David
+- AI(David): Review the 1st video demo by Jim
+- AI(Jim): Open issues that cover the following topics/questions to get answers/shared understanding
+  - 1) What are the main differences (scope, features, ) between TestGround, TestLab, ipfs/benchmarks?
+  - 2) ElasticSearch Review
```

Comment:
@jimpick did you get the chance to write this one up?

Reply:
Going to do that now... (Canadian Thanksgiving holiday here, different date than USA, so busy time)

Reply:
Understood :) Happy Thanksgiving!

Reply:
Here's an initial issue for ElasticSearch: https://github.com/ipfs/testground/issues/67